Code/Framework/AzCore/Tests/Memory/AllocatorBenchmarks.cpp

```diff
@@ -259,6 +261,7 @@ namespace Benchmark
     public:
         void Benchmark(benchmark::State& state)
         {
+            m_allocator = AZStd::make_unique<TestAllocatorType>();
```

Comment:
It seems that for consistency, this RecordedAllocationBenchmarkFixture class should override `SetUp` and `TearDown` and create the allocator the same way it is done as in the `AllocatorBenchmarkFixture`

Reply:
Fixed.